OptimoveSDK/Sources/Classes/Optimove.swift

```diff
@@ -26,6 +26,10 @@ typealias Logger = OptimoveCore.Logger
 
     private override init() {
         self.container = Assembly().makeContainer()
+        container.resolve { serviceLocator in
```

Comment:
Based on my current understanding of the container, this change should be OK.

Perhaps the logger init could stay inside configure(for:) but I wasn't 100% sure and the logger doesn't seem to be able to init more than once so I moved it.

Visitor ID generation is quite simple and is safe to be run multiple times so it could have remained in both places, but I didn't see a need to duplicate the work.